docs/installing-packages.md

```diff
@@ -0,0 +1,203 @@
+# Installing packages
+This documentation dives deeper into how a user can configure packages for installation inside the EIB image. Furthermore, it explains the **RPM resolution** process that EIB goes through so it can ensure that configured packages will be successfully installed even in an **air-gapped** environment.
+
+Below you can find a table of contents for easier navigation:
+
+1. [Supported systems](#supported-systems)
+2. [Configure packages for installation](#configure-packages-for-installation)
+    * [Install packages through 'packageList'](#install-packages-through-packagelist)
+        * [Install a package from a third-party repo](#install-a-package-from-a-third-party-repo)
+        * [Install a package from SUSE's internal repositories](#install-a-package-from-suses-internal-repositories)
+    * [Side-load RPMs](#side-load-rpms)
+        * [RPM with dependency resolution from a third-party repository](#rpm-with-dependency-resolution-from-a-third-party-repository)
+        * [RPM with depdendency resolution from SUSE's internal repositories](#rpm-with-depdendency-resolution-from-suses-internal-repositories)
+    * [Installing unsigned packages](#installing-unsigned-packages)
+3. [Package installation workflow](#package-installation-workflow)
+    * [Running the EIB container](#running-the-eib-container)
+    * [Building the EIB image](#building-the-eib-image)
+        * [RPM resolution process](#rpm-resolution-process)    
+        * [Troubleshooting](#troubleshooting)
+    * [Booting the EIB image](#booting-the-eib-image)
+
+## Supported systems
+EIB's **RPM resolution** process and package installation has been tested on the following `x86_64` systems: 
+1. [Ubuntu 22.04](https://releases.ubuntu.com/jammy/)
+2. [SLES 15-SP5](https://www.suse.com/download/sles/)
+3. [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed/)
+4. [Fedora Linux](https://fedoraproject.org/server/download)
+
+## Configure packages for installation
```

Comment:
I don't think "configure" is the right word to use here. You're simply "specifying" which packages to install. To me, "configure" means we're providing specific information to manipulate a package on a per package basis.